domains/tim.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Tim is a Dev",
+  "owner": {
+    "username": "xorob0",
+    "email": "xorob0@toum"
+  },
+  "repo": "https://github.com/xorob0/register",
+  "record": {
+    "URL": "https://github.com/tommitchelmore",
+    "CNAME": "https://toum.me"
```

This is not a valid cname

Maybe this is what you wanted to do?
```suggestion
    "CNAME": "toum.me"
```